net/autossh/src/etc/rc.d/autossh

```diff
@@ -0,0 +1,182 @@
+#!/bin/sh
```

Comment:
Normally whene there's no init script it's called opnsense-xxx so there wont be a collision when a new script comes in future by pkg maintainer

Reply:
Ahh - I recall wondering about this - thanks, and sorting it out now

Reply:
Also dont forget in rc.conf.d .. 👍